### PR TITLE
Add the 2021-09-23 pipeline

### PR DESCRIPTION
**Pull request by alexwlchan**

### CODE REVIEW

File: infrastructure/modules/worker/main.tf [hunk 1]
```diff
@@ -1,13 +1,15 @@
 locals {
   # Override the default service name if requested
   deployment_service_name = var.deployment_service_name == "" ? var.name : var.deployment_service_name
+
+  service_name = var.service_name == "" ? var.name : var.service_name
```
Comment (alicefuzier) [inline infrastructure/modules/worker/main.tf:5]: I'm struggling to understand this, or the other related changes. Would it make sense to add a comment explaining?

Comment (alexwlchan) [reply infrastructure/modules/worker/main.tf:12]: I've added a comment, any better?

Comment (alicefuzier) [reply infrastructure/modules/worker/main.tf:12]: Sort of.. doesn't that make it clash with other pipelines though?

Comment (alexwlchan) [reply infrastructure/modules/worker/main.tf:12]: No:

* In the ECS console, service names only have to be unique within a cluster. You can repeat names across clusters.
* In the logging cluster, you can filter by (cluster, service name) to find just those services – but now you can actually find the service name.

File: reindexer/get_reindex_status.py [hunk 2]
```diff
@@ -242,6 +275,21 @@ def main(reindex_date):
...
+        rows = [
+            [q_url.split("/")[-1].replace(f"catalogue-{reindex_date}_", ""), size]
+            for q_url, size in sorted(non_empty_dlqs.items())
+        ]
+        print(tabulate.tabulate(rows, tablefmt="plain", colalign=("left", "right")))
+    else:
+        print("There are no messages on DLQs")
+
+    print("")
+
```
Comment (alexwlchan) [inline reindexer/get_reindex_status.py:292]: This change is to highlight any DLQs in this pipeline that have failing messages:

Screenshot 2021-09-24 at 10 15 15

**Review by alicefuzier — APPROVED** [review]

**Pull request merged**